### ISSUE TITLE
{Freeway-Bug} [Adaptive Cards-Left Navigation]: Luminosity ratio of selected 'Agenda.json' control present on left navigation with respect to its non-selected adjacent background color area is 1.71:1, which is less than minimum required ratio i.e., 3:1.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9216]
### Target Platforms

UWP

### SDK Version

1.0.3.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Visualizer
OS Version: 25H2 (OS Build 26200.5670) 
App Version: 1.0.3.0
Tool: Color Contrast analyzer -> Launch through Accessibility Insights for Windows Tool

**Repro steps:**
1. Open the Adaptive Cards Visualizer application.
2. Navigate to the 'Agenda.json' control present on left navigation using tab key and activate it using enter key.
3. Launch Accessibility Insights for Windows tool and open Color contrast analyzer and select the foreground and its adjacent background color of the selected 'Agenda.json' control present on left navigation with respect to its non-selected adjacent background.
4. Verify that, luminosity ratio of selected 'Agenda.json' control present on left navigation with respect to its non-selected adjacent background color area is 1.71:1, which is less than minimum required ratio i.e., 3:1 or not.

**Actual Result:**
Luminosity ratio of selected 'Agenda.json' control present on left navigation with respect to its non-selected adjacent background color area is 1.71:1, which is less than minimum required ratio i.e., 3:1.

**Note:**
Same issue replicates for all the left navigation control if selected.

**Expected Result:**
Luminosity ratio of selected 'Agenda.json' control present on left navigation with respect to its non-selected adjacent background color area should be greater than or equal to the minimum required ratio i.e., 3:1.

**User Impact:**
Users with visual impairments, especially those with low vision or color blindness, may struggle to distinguish between the selected and non-selected controls due to the insufficient contrast due to which users might not distinguish the selected and non-selected control.

**WCAG Reference link:**
https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![Image](https://github.com/user-attachments/assets/dca2b874-2566-4a38-9a0b-d9102b558aff)

### Card JSON

```JSON
N/A
```
